File: BlueSkyTaskThree - java version/src/main/java/com/bluesky/model/destination/Group.java
package com.bluesky.model.destination;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;


@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Group {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private String gid;
    private String name;
    private String description;
    private String city;
    private String state;


}

File: BlueSkyTaskThree - kotlin version/src/main/kotlin/com/bluesky/model/destination/Group.kt
package com.bluesky.model.destination

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
class Group {


    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var gid: String = ""
    var name: String = ""
    var description: String = ""
    var city: String = ""
    var state: String = ""
}